src/listen.ts

```diff
@@ -124,11 +141,15 @@ export async function listen(
     await tunnel?.close().catch(() => {});
   };
 
+  function onRejected() {
+    listhenOptions.clipboard = false;
+  }
+
   if (listhenOptions.clipboard) {
     const clipboardy = await import("clipboardy").then((r) => r.default || r);
-    await clipboardy.write(getURL()).catch(() => {
-      listhenOptions.clipboard = false;
-    });
+    await (listhenOptions.ipc
+      ? clipboardy.write(ipcSocket).catch(onRejected)
+      : clipboardy.write(getURL()).catch(onRejected));
```

Comment:
Maybe `getURL` can also be aware of ipcSocket to return it's path when called too?

Reply:
`getURLs` is now also be aware of ipcSocket and will return immediately, if it's present

Reply:
I miss interpreted this comment. `getURL` will also be aware of the ipcSocket with the next push

Reply:
Now both functions (`getURL` and `getURLs`) are handling the IPC socket path.